internal/crypt/crypt_impl.go

```diff
@@ -1,3 +1,5 @@
+// +build !macos
```

Comment:
What is `+build !macos`?

Reply:
Conditional compilation for all non-macos systems.